pkg/nsx/services/common/init.go

```diff
@@ -30,3 +31,8 @@ func init() {
 	Converter = bindings.NewTypeConverter()
 	Converter.SetMode(bindings.REST)
 }
+
+var (
+	AOS = pointy.String // address of string
+	AOI = pointy.Int64  // address of int
```

Comment:
How about directly using `String` and `Int64` as the aliases, like what [aws sdk](https://github.com/aws/aws-sdk-go/blob/9d10b7469ebfe21f9ab825461b034f5ac6fc4b8b/aws/convert_types.go#L123) did? We may need not only int64, but also int, int32, etc.. The name like AOI contains too little info about the target type.

Reply:
OK